config/eslint/base/index.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
     '**/playwright-report/**',
   ],
   rules: {
-    '@typescript-eslint/no-explicit-any': 'warn',
+    '@typescript-eslint/no-explicit-any': 'off', // rome is enough
```

Comment:
What about webstorm users?

Reply:
> What about webstorm users?

does rome work for you? they both lint the same thing, having both on is unnecessary